Motorcycle.js

```diff
@@ -1,6 +1,23 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
+function Motorcycle(seats, running) {
```

Comment:
Here, we probably want to inherit other properties from car, such as make, model, etc.
```
function Motorcycle(make, model, year, color, seats, passengers) {
	Car.call(this, make, model, year, color, seats, passengers);
```